Makefile

```diff
@@ -15,6 +15,7 @@ vet:
 mocks:
 	mockgen -package mocks -destination go/mocks/filtering.go github.com/rode/grafeas-elasticsearch/go/v1beta1/storage/filtering Filterer
 	mockgen -package mocks -destination go/mocks/index.go github.com/rode/grafeas-elasticsearch/go/v1beta1/storage/esutil IndexManager
+	counterfeiter go/v1beta1/storage/esutil Client
 	mockgen -package mocks -destination go/mocks/orchestrator.go github.com/rode/grafeas-elasticsearch/go/v1beta1/storage/migration Orchestrator
```

Comment:
Also we should probably make an issue to move everything over to `counterfeiter` in Rode and `greafeas-elasticsearch`